addons/account/models/account_move_line.py

```diff
@@ -598,6 +598,7 @@ def _compute_account_id(self):
                     line.account_id = previous_two_accounts
                 else:
                     line.account_id = line.move_id.journal_id.default_account_id
+        self._check_constrains_account_id_journal_id()
```

Comment:
Do we need this?
Isn't an assignation on the field triggering the write on the field and thus the write on account.move and thus the constrains?